lib/rake/circle.rake

```diff
@@ -187,10 +187,8 @@ def start_sauce_connect
   # pin the version we use to the last working version, while we schedule the task to get the upgraded version
   # working. You can do this by replacing `sc_download_url` with a hard-coded download url.
 
-  # Temporarily pinning sauceconnect version to 4.5.4 since 4.6.0 seems to have broken us.
-  #sc_version_info = JSON.parse(Net::HTTP.get(URI('https://saucelabs.com/versions.json')))
-  #sc_download_url = sc_version_info['Sauce Connect']['linux']['download_url']
-  sc_download_url = 'https://saucelabs.com/downloads/sc-4.5.4-linux.tar.gz'
+  sc_version_info = JSON.parse(Net::HTTP.get(URI('https://saucelabs.com/versions.json')))
+  sc_download_url = sc_version_info['Sauce Connect']['linux']['download_url']
```

Comment:
This is a neat approach.  I guess the pro is that we are always using latest, and will fail if/when a newer version breaks us, while the con is that exactly that happens (versus pinning to a known-good version).

Reply:
Yea, this is the code we'd had before my "temporary" change to pin the version, since we did get broken. I think we later decided to deprioritize unpinning it since we didn't have a specific reason to then.